diff -up kcoreaddons-5.85.0/src/lib/io/knetworkmounts.h.omv~ kcoreaddons-5.85.0/src/lib/io/knetworkmounts.h
--- kcoreaddons-5.85.0/src/lib/io/knetworkmounts.h.omv~	2021-08-06 19:52:50.325060047 +0200
+++ kcoreaddons-5.85.0/src/lib/io/knetworkmounts.h	2021-08-06 19:54:20.280518495 +0200
@@ -275,9 +275,11 @@ private:
     /// Creates a new KNetworkMounts object
     KNetworkMounts();
 
+public:
     /// Destructor
     ~KNetworkMounts();
 
+private:
     std::unique_ptr<class KNetworkMountsPrivate> const d;
 };
 
