--- kcoreaddons-5.0.0/src/lib/io/kfilesystemtype.cpp.omv~	2014-07-11 14:51:23.408208055 +0200
+++ kcoreaddons-5.0.0/src/lib/io/kfilesystemtype.cpp	2014-07-11 14:51:38.791015743 +0200
@@ -99,7 +99,7 @@ KFileSystemType::Type determineFileSyste
         //qDebug() << path << errno << strerror(errno);
         return KFileSystemType::Unknown;
     }
-    switch (buf.f_type) {
+    switch ((uint32_t)buf.f_type) {
     case NFS_SUPER_MAGIC:
     case AUTOFS_SUPER_MAGIC:
     case AUTOFSNG_SUPER_MAGIC:
